uvicorn/protocols/http/h11_impl.py

```diff
@@ -68,6 +79,7 @@ def __init__(
         self,
         config: Config,
         server_state: ServerState,
+        app_state: Dict[str, Any],
```

Comment:
Since we already copied (via `.copy()`) should we rename this to `request_state` or something like that? To make it clear via the name that it is a copy of the app state.

Reply:
I prefer `app_state`. :+1:

Reply:
Maybe `copied_app_state`? No biggie if you prefer to just keep `app_state`

Reply:
Just `app_state` is fine. :pray: